src/wiki/ventoy.md

```diff
@@ -0,0 +1,106 @@
+# Creating a Multi-OS Bootable Drive (with Ventoy)
+In this article, you will learn how to create a multi-OS bootable drive. 
+
+## Creating the Bootable Drive
+This section will teach you how to create a bootable drive for your needs.
+1. Download Ventoy [here](https://www.ventoy.net/en/download.html). This is the tool that we will be using to create a bootable drive.
+::: Warning
+2. Plug in a USB drive that has at least 8GB.
+3. Right click on the file that you have just downloaded and select extract. Then, enter the folder named `ventoy-1.0.xx` and open an application named `Ventoy2Disk.exe`. A UAC prompt will pop-up, click yes.
+4. Select a drive in the device selection section. This will be your bootable drive. If you can't see the drive you want to use, press the green button to refresh the list.
+In this area, you will see how to create your USB for being bootable.
+1. [Download the Ventoy application that will help us to make the USB being capable with various ISOs.](https://www.ventoy.net/en/download.html)
+![Download page of Gparted](img/ventoy/Gparted.png)
+6. Click install, and Ventoy will create a bootable drive for you. After it finishes, you will see a drive named as "Ventoy", this is where you will place your ISO files for the bootable drive.
+3. For this time, we will use the Ventoy2Disk.exe, we click on it, and we click to “yes” in the UAC screen that pooped.
+4. Then at devices section select your USB that you want to make a Ventoy or if you can't see it press the green button to refresh the list.
+## Downloading Windows to the Bootable Drive
+### Downloading the ISO file
+![Menu for selecting settings in Ventoy](<img/ventoy/Secure boot and GPT.png>)
+6. At the last step, click on install and wait some seconds to get the job done.
```

Comment:
What is happening here, why are there more steps of the installation here? Shouldn't this screenshot be on line 14?